pkg/controller/petset/pet.go

```diff
@@ -169,12 +169,14 @@ func (p *apiServerPetClient) Get(pet *pcb) (*pcb, bool, error) {
 	found := true
```

Comment:
Why not just get rid of `found` at this point? Isn't this equivalent to:
```go
	ns := pet.parent.Namespace
	pod, err := podClient(p.c, ns).Get(pet.pod.Name)
	if errors.IsNotFound(err) {
		return nil, false, nil
	}
	if err != nil {
		return nil, false, err
	}
	realPet := *pet
	realPet.pod = pod
	return &realPet, true, nil
```
which seems even simpler